mibidata/mibi_image.py

```diff
@@ -169,6 +169,7 @@ def point_name(self, value):
         warnings.warn('The "point_name" attribute is deprecated. '
                       'Setting "fov_name" to "{}".'.format(value))
         self.fov_name = value  # pylint: disable=attribute-defined-outside-init
+        self._user_defined_attributes.remove('point_name')
```

Comment:
This will raise a ValueError is a user tries to set `point_name` on an existing MibiImage. I am going to remove it as part of a cleanup commit that also fixes some docstring formatting.

Reply:
Oh, ok, yes I realized later I should have used a try-except block in the setter. Anyways, no need for it now.